### ISSUE TITLE
profiler.takePreciseCoverage() is Failing with Chrome 78.0.3904.87

### ISSUE THREAD

**Issue by SergeyPirogov** [kklisura/chrome-devtools-java-client#31]
Chrome version 78.0.3904.87.

Start failing when 

List<ScriptCoverage> scriptCoverages = profiler.takePreciseCoverage()

```
com.github.kklisura.cdt.services.exceptions.ChromeDevToolsInvocationException: Failed sending web socket message.
	at com.github.kklisura.cdt.services.impl.ChromeDevToolsServiceImpl.invoke(ChromeDevToolsServiceImpl.java:175)
	at com.github.kklisura.cdt.services.invocation.CommandInvocationHandler.invoke(CommandInvocationHandler.java:87)
```

**Comment by SergeyPirogov** [comment]
@It's a problem related to Burrer overflow. @kklisura I see that fix is already in master. When it will be released

**Comment by kklisura** [comment]
Released as `2.1.0`.
